update/Pos/Update/Poll/Types.hs

```diff
@@ -211,6 +213,9 @@ data PollModifier = PollModifier
     , pmActiveProps       :: !(MapModifier UpId ProposalState)
     , pmDelActivePropsIdx :: !(HashMap ApplicationName (HashSet UpId))
     , pmSlottingData      :: !(Maybe SlottingData)
+    , pmEpochProposers    :: !(Maybe (HashSet StakeholderId))
+    -- ^ Cache of DB state.
```

Comment:
because it will be kolbasa like
```
runToilTLocal
    :: (Functor m)
    => UtxoModifier
    -> MemPool
    -> UndoMap
    -> ToilT () m a
    -> m (a, ToilModifier)
```
but ok, I'll do. не получилось проскочить мимо громака....